src/applications/disability-benefits/all-claims/pages/addDisabilities.js

```diff
@@ -44,43 +35,31 @@ export const uiSchema = {
       ),
       'view:descriptionInfo': {
         'ui:description': () => (
-          <AlertBox isVisible status="info">
-            <p>
-              Below are some details that may be helpful to include when
-              describing your disability:
-            </p>
-            <ul>
-              <li>The part of your body that's affected</li>
-              <li>
-                If your disability is on the right side or left side of your
-                body
-              </li>
-              <li>The part of your body that isn't working right</li>
-            </ul>
-          </AlertBox>
+          <div>
+            <p>For example, foot pain, back pain, or hearing loss.</p>
+            <AdditionalInfo triggerText="What if I don't know the name of my condition?">
```

Comment:
Looks like maybe the wrong kind of apostrophe here

Reply:
:male_detective: